tests/integration/pilot/gateway_test.go

```diff
@@ -668,7 +668,7 @@ func TestGatewayMetricsEndpoints(t *testing.T) {
 		Features("traffic.gateway.metrics").
 		Run(func(t framework.TestContext) {
 			c := t.Clusters().Default()
-			podIPs, err := i.PodIPsFor(c, "app=istio-ingressgateway")
+			podIPs, err := i.PodIPsFor(c, "istio-system", "app=istio-ingressgateway")
```

Comment:
Could the `istio-system` namespace be replaced with `i.cfg.SystemNamespace` like above?

Reply:
Fixed this